scripts/ingest_dfc/dfc/family.py

```diff
@@ -185,7 +197,7 @@ def _add_family_document_slug(
     """
     family_document_slug = Slug(
         name=row.cpr_document_slug,
-        family_document_id=family_document.physical_document_id,
+        family_document_import_id=family_document.import_id,
```

Comment:
@THOR300 - if you do need to get things working this is the line you'll need to change